public/resources/remix/index.js

```diff
@@ -6,122 +6,137 @@
+    analyticsHtml.type = "text/javascript";
+    analyticsHtml.innerHTML = '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' + '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' + 'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' + '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' + 'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' + 'ga(\'send\', \'pageview\');\n';
+    head.appendChild(analyticsHtml);
   }
 
-  function setupBar(err, $) {
+  function setupBar(err) {
     if(err) {
       console.log("[Thimble Error] Unable to inject Remix UI. Error was: `" + err + "`");
       return;
     }
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
-    var detailsBar = $(".details-bar");
-    detailsBar.attr("style", "");
+    var detailsBar = document.querySelector(".details-bar");
+    detailsBar.setAttribute("style", "");
 
     if(isTouchDevice) {
-      detailsBar.addClass("touch-mode");
-    } else {
-      detailsBar.addClass("mouse-mode");
+      detailsBar.classList.add("touch-mode");
+    }
+    else {
+      detailsBar.classList.add("mouse-mode");
     }
 
-    detailsBar.on("click", ".thimble-button",function(){
-      detailsBar.removeClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.className == "thimble-button") {
+        detailsBar.classList.remove("collapsed");
+        return false;
+      }
     });
 
-    detailsBar.on("click", ".close-details-bar", function(){
-      detailsBar.addClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.className == "close-details-bar") {
+        detailsBar.classList.add("collapsed");
+        return false;
+      }
     });
 
-    $(".mouse-mode").on("mouseenter",function(){
-      detailsBar.removeClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseenter", function(){
+      detailsBar.classList.remove("collapsed");
     });
 
-    $(".mouse-mode").on("mouseleave",function(){
-      detailsBar.addClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseleave", function(){
+      detailsBar.classList.add("collapsed");
     });
   }
 
-  function injectDetailsBar($, metadata, callback) {
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.responseType = "text";
+    var url = metadata.host + "/projects/remix-bar";
```

Comment:
@cadecairos, @gideonthomas  what's the right way for us to do XHR from publish to a remix page hosted on thimbleprojects cross domain?